core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -864,6 +864,18 @@
   public static final String CARBON_MERGE_SORT_PREFETCH = "carbon.merge.sort.prefetch";
   public static final String CARBON_MERGE_SORT_PREFETCH_DEFAULT = "true";
 
+  /**
+   * to determine to use the rdd persist or not for insert query.
```

Comment:
Please write clear comment, when this should be used and what is the impact if not used, why default value is false?

Reply:
done